RusTaxi/Classes/Screens/VC/SlideShow Controller/MainController.swift

```diff
@@ -32,6 +32,16 @@ class MainController: UIViewController, MKMapViewDelegate, CLLocationManagerDele
 		initializeLocationManager()
 		registerNibs()
 		initializeFirstAddressCells()
+		animatingView()
+	}
+	
+	override func viewDidLayoutSubviews() {
+		super.viewDidLayoutSubviews()
+		if acceptView == nil {
+			acceptView = Bundle.main.loadNibNamed("AcceptView", owner: self, options: nil)?.first as? AcceptView
+			acceptView?.frame = CGRect(x: 0, y: 0, width: 0, height: 0)
+			self.view.addSubview(acceptView!)
```

Comment:
view.addSubview(acceptView!) must be in `viewDidLoad`

but `.frame` must be stay here in `viewDidLayoutSubviews`

Reply:
and forget about `== nil` condition. we are using only `if let` or `guard let`